README.md

```diff
@@ -42,25 +32,33 @@ So that I only see the relevant tasks
 As a person who doesn't like counting by hand
 I want to see a total number of tasks
 So that I don't have to count
-
-As someone who has done lots of stuff
-I want to be able to clear my completed tasks
-So I never see them again
 ```
 
-As you may imagine, implementing a To-do list is very much a solved problem. However, we are mainly interested in seeing how you approach testing and design. We are looking for:
+##MVP Roadmap##
+
+- MVP1 - Lists tasks added.
+- MVP2 - Ability to mark tasks as done.
+- MVP3 - MVP3 + styling
+- MVP4 - Filter tasks by "All", "Active" & "Completed"
+- MVP5 - Display total number of tasks
+- 
+##Deployed Version##
+A deployed version can be played with [here](https://blooming-bastion-6250.herokuapp.com/). This is being served by a node express server, but does not yet have any persistent storage.
```

Comment:
Maybe pop this up in the description so others can immediately see a deployed version